### DISCUSSION TITLE
[Skia]How can I remove the default  context menu entry of a TextBox control  when right clicking on a TextBox

### DISCUSSION THREAD

**Discussion by sushuying521** [unoplatform/uno#15587] [General]
[Skia]How can I remove the default  context menu entry of a TextBox control  when right clicking on a TextBox
<PackageReference Include="SkiaSharp" Version="2.88.3" />
![lQDPKHxEhds1Xm_NBLXNA_6wBid9qWlRP0oFxXxzHnv9AA_1022_1205](https://github.com/unoplatform/uno/assets/90383758/43c30396-88f1-4b19-b99c-f9ea5cc50e6e)

**Comment by sushuying521** [comment]
<PackageReference Include="SkiaSharp" Version="2.88.3" />

**Comment by mcNets** [comment]
Maybe I'm wrong but I think you can't, even if you set the ContentFlyout to null, the control generates a new context menu when right tapped.

https://github.com/unoplatform/uno/blob/ee6392f1c0099b7434b417fd4c3eb488f8ade3e8/src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.skia.cs#L824-L842

**Reply by jeromelaban** [reply]
@ramezgerges would you know if this is possible on Windows?

**Reply by ramezgerges** [reply]
There seems to be a `ContextMenuOpening` event that can be `Handled` to prevent the context menu from opening, but it's currently not implemented in Uno.
